unit-tests/algo/d2rgb/test-frame-drop.py

```diff
@@ -0,0 +1,88 @@
+import pyrealsense2 as rs, common as test, ac
+
+# This variable controls how many calibrations we do while testing for frame drops
+n_cal = 5
+
+# We set the environment variables to suit this test
+test.set_env_vars({"RS2_AC_DISABLE_CONDITIONS":"1",
+                   "RS2_AC_DISABLE_RETRIES":"1",
+                   "RS2_AC_FORCE_BAD_RESULT":"1"
+                   })
```

Comment:
What about LOG_TO_STDOUT?

Reply:
Do we want to do that for each test?

Reply:
If you look at the log, it'd be nice to see it without having to rerun

Reply:
If you look at the log, it'd be nice to see it without having to rerun

Reply:
But the tests usually log to some file. Won't this make them log to the monitor?